projectroles/views.py

```diff
@@ -1648,7 +1647,7 @@ def delete_assignment(self, request, instance):
         # Delete object itself
         instance.delete()
         # Remove project star from user if it exists
-        remove_tag(project=project, user=user)
+        set_tag_state(project, user, star=False)
```

Comment:
We should not set this setting to false here. Instead, we should delete it using `app_settings.delete()`.

In fact, we should probably delete all `PROJECT-USER` scope settings for the project & user combination when a user's role is removed from a project. But that's a separate issue, I'll think about it and open a ticket..